plugins/modules/notification/slack.py

```diff
@@ -206,8 +231,24 @@ def escape_quotes(text):
     return "".join(escape_table.get(c, c) for c in text)
 
 
+def recursive_escape_quotes(obj, keys):
+    '''Recursively escape quotes inside supplied keys inside block kit objects'''
+    if isinstance(obj, dict):
+        escaped = {}
+        for k, v in obj.items():
+            if isinstance(v, str) and k in keys:
+                escaped[k] = escape_quotes(v)
+            else:
+                escaped[k] = recursive_escape_quotes(v, keys)
+    elif isinstance(obj, list):
+        escaped = [recursive_escape_quotes(v, keys) for v in obj]
+    else:
+        return obj
```

Comment:
```suggestion
        escaped = obj
```
Alternatively, `return` in every `if`/`else` branch.